### ISSUE TITLE
energy_left and total_pack_energy is gone from the API

### ISSUE THREAD

**Issue by gak** [gak/teslatte#22]
```
called `Result::unwrap()` on an `Err` value: GET https://owner-api.teslamotors.com/api/1/products json decode error

Caused by:
    data did not match any variant of untagged enum Product at line 1 column 7061
```

JSON omitted due to privacy concerns. Will investigate.

**Comment by gak** [comment]
`energy_left` and `total_pack_energy` is gone from the API.

**Comment by gak** [comment]
Fixed in 1e6c1d5ff3470ce497243bbaec37c475163e61b2
